pycqed/analysis_v2/spectroscopy_analysis.py

```diff
@@ -13,8 +13,11 @@
 from pycqed.analysis import fitting_models as fit_mods
 import lmfit
 
+from importlib import reload  # Useful for reloading while testing
```

Comment:
Please remove debugging stuff from here.

Reply:
Removed all the reload commands, also to reload the base_class and the fitting_mods